src/main/java/game/ResourceLoader.java

```diff
@@ -45,7 +45,7 @@ public AudioClip getSound(String name) {
 			url = getClass().getClassLoader().getResource("res/" + name);
 			sound = Applet.newAudioClip(url);
 			sounds.put(name,sound);		
-			LOGGER.trace("Loaded sound file: " + name);
+//			LOGGER.trace("Loaded sound file: " + name);
 		} catch (Exception e) {
 			LOGGER.error("Cound not locate sound " + name + ": " + e.getMessage());
```

Comment:
this LOGGER.error call will be removed later and handled in the Exception Handling aspect right?

Reply:
Yep. Anything in a `catch` block should be handled by the exception-handling aspect. So it's not part of this story specifically.